client/src/Components/Header/Header.scss

```diff
@@ -0,0 +1,24 @@
+@import "../../Utilities//SCSS-Variables//variables.scss";
+
+.header {
+  display: flex;
```

Comment:
remove "header" class from header html node. Remove period(.) before header css rule here, and it will still work

Reply:
Yeah, fixed the changes